packages/next/src/client/components/router-reducer/create-initial-router-state.ts

```diff
@@ -102,7 +102,13 @@ export function createInitialRouterState({
     createPrefetchCacheEntryForInitialLoad({
       url,
       kind: PrefetchKind.AUTO,
-      data: { f: initialFlightData, c: undefined, i: !!couldBeIntercepted },
+      data: {
+        f: initialFlightData,
+        c: undefined,
+        i: !!couldBeIntercepted,
+        // TODO: the server should probably send a value for this. Default to false for now.
+        p: false,
```

Comment:
I'm going to follow up with this separately as I don't think it's a blocker - it just means the seeded prefetch for the page that gets initially loaded won't be marked as a "full" prefetch if its static.